common/test/common/commercial/hosted/ColourTest.scala

```diff
@@ -0,0 +1,45 @@
+package common.commercial.hosted
+
+import org.scalatest.{FlatSpec, Matchers}
+
+/*
+ * For colours, see http://www.colorhexa.com/<hexcode>
+ * eg. http://www.colorhexa.com/2ec869
+ */
+class ColourTest extends FlatSpec with Matchers {
+
+  "isDark" should "be true for Zootropolis green" in {
+    val zootropolisColour = Colour("#2ec869")
+    zootropolisColour shouldBe 'dark
```

Comment:
Yes. http://www.scalatest.org/user_guide/using_matchers#checkingBooleanPropertiesWithBe

(But if it makes the test worse, I'll get rid of it.)